lib/galaxy/util/oset.py

```diff
@@ -2,61 +2,70 @@
 Ordered set implementation from https://code.activestate.com/recipes/576694/
 """
 from collections.abc import MutableSet
+from typing import 
+from typing import (
+    Any,
+    Dict,
+    List,
+    TYPE_CHECKING,
+)
 
+if TYPE_CHECKING:
+    from bioblend.galaxy import GalaxyInstance
 
 class OrderedSet(MutableSet):
-    def __init__(self, iterable=None):
+    def __init__(self, iterable=None) -> List[Dict[str, Any]]:
```

Comment:
this should be `-> None` (since it's `__init__`)
Also, iterable is, most likely, `Iterable` (imported from `typing`)